### PR TITLE
add reference to gatekeeper repo

### PR DESCRIPTION
**Pull request by ch-stark**
creating a link between gatekeeper/rego repo and policy-collection repo

### CODE REVIEW

File: community/CM-Configuration-Management/policy-resiliency-image-pruner.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+apiVersion: policy.open-cluster-management.io/v1
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-resiliency-image-pruner.yaml:1]: can you add this policy to readme?

File: community/CM-Configuration-Management/policy-resiliency-image-pruner.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+kind: Policy
+metadata:
+  name: policy-resiliency-image-pruner
+  annotations:
+    policy.open-cluster-management.io/standards: NIST-CSF
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-resiliency-image-pruner.yaml:6]: can you change the standards/categories/controls to use NIST 800-53

File: community/CM-Configuration-Management/policy-gatekeeper-monitor-clusteroperator.yaml [hunk 3]
```diff
@@ -0,0 +1,121 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-gatekeeper-monitor-clusteroperator
+  namespace: rhacm-policies
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-gatekeeper-monitor-clusteroperator.yaml]: remove namespace

File: community/CM-Configuration-Management/policy-gatekeeper-monitor-clusteroperator.yaml [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: constraints.gatekeeper.sh/v1beta1
+                kind: K8sMonitorClusterOperator
+                metadata:
+                  name: monitor-clusteroperator
+                status:
+                  totalViolations: 0
+    - objectDefinition:
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-gatekeeper-monitor-clusteroperator.yaml]: admission check doesn't make sense for this policy

File: community/CM-Configuration-Management/policy-gatekeeper-monitor-clusteroperator.yaml [hunk 5]
```diff
@@ -0,0 +1,121 @@
+apiVersion: policy.open-cluster-management.io/v1
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-gatekeeper-monitor-clusteroperator.yaml]: can you add this policy to readme?

File: community/CM-Configuration-Management/policy-configure-subscription-admin-hub.yaml [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+kind: PlacementRule
+metadata:
+  name: placement-policy-configure-subscription-admin-hub
+spec:
+  clusterConditions:
+  - status: "True"
+    type: ManagedClusterConditionAvailable
+  clusterSelector:
+    matchExpressions:
+      - {key: name, operator: In, values: ["local-cluster"]}
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-configure-subscription-admin-hub.yaml:94]: can you change the cluster selector to match other policies?

File: community/CM-Configuration-Management/policy-checkclusteroperator.yaml [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+metadata:
+  name: placement-policy-checkclusteroperator
+  namespace: 
+spec:
+  clusterConditions:
+  - status: "True"
+    type: ManagedClusterConditionAvailable
+  clusterSelector:
+    matchExpressions:
+      - {key: name, operator: In, values: ["local-cluster"]}
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-checkclusteroperator.yaml:84]: can you change the cluster selector to match other policies?

File: community/CM-Configuration-Management/policy-checkclusteroperator.yaml [hunk 8]
```diff
@@ -0,0 +1,84 @@
+apiVersion: policy.open-cluster-management.io/v1
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-checkclusteroperator.yaml:1]: can you add this policy to readme?

File: community/CM-Configuration-Management/policy-audit-logging-sample.yaml [hunk 9]
```diff
@@ -0,0 +1,55 @@
+apiVersion: policy.open-cluster-management.io/v1
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-audit-logging-sample.yaml:1]: can you add this policy to readme?

File: community/CM-Configuration-Management/policy-audit-logging-sample.yaml [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+  apiGroup: policy.open-cluster-management.io
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+  name: placement-policy-audit
+spec:
+  clusterConditions:
+  - status: "True"
+    type: ManagedClusterConditionAvailable
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-audit-logging-sample.yaml:55]: can you add `clusterSelector`?

File: community/CM-Configuration-Management/policy-resiliency-image-pruner.yaml [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+kind: PlacementRule
+metadata:
+  name: placement-policy-resiliency-image-pruner
+spec:
+  clusterConditions:
+  - status: "True"
+    type: ManagedClusterConditionAvailable
+  clusterSelector:
+    matchExpressions:
+      - {key: local-cluster, operator: In, values: ["true"]}
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-resiliency-image-pruner.yaml:69]: can you change the cluster selector to match other policies?

**Comment by ycao56** [comment]
@ch-stark please also sign off all commits. You can run `git rebase --signoff main` and then `git push --force`

**Comment by ch-stark** [comment]
have issues with signing existing commits hence will open other PR and work in all comments

**Pull request closed without merging**